src/i18n/en/organizations.i18n.json

```
@@ -0,0 +1,16 @@
+{
+	"organizations": "Organizations",
+	"fetchOrganizationsFailedError": "Organizations fetch failed - error: {{error}}",
+	"noOrganizationsFound": "No organizations found",
+	"noOrganizationsFoundExtended": "No organizations found, error: {{error}}",
+	"noOrganizationsChosen": "No organizations chosen, you must pick organization, reload your workspace and pick organization",
+	"noOrganizationsConfigured": "User organization not configured, please select your organization in the picker above",
+	"userNotFound": "Invalid token, user not found, validate your token and reload the workspace",
+	"userNotFoundExtended": "Invalid token, user not found, validate your token and reload the workspace, error: {{error}}",
+	"tokenNotFound": "API token not found",
+	"missingOrganizationId": "Missing organization ID",
```

Where do you use it?
```suggestion
	"tokenNotFound": "API token not found",
	"missingOrganizationId": "Missing organization ID",
```

---

removed